test/test.tsx

```diff
@@ -6,3 +6,4 @@ import { Nav } from "@patternfly/react-core";
 
 import { Wizard } from "@patternfly/react-core";
 import { WizardFooter } from "@patternfly/react-core/next";
+import { ApplicationLauncher } from "@patternfly/react-core";
```

Comment:
We might want to alphabetize this too

Reply:
Rearranged that whole file to put all imports alphabetically at the top